src/site/resources/css/site.css

```diff
@@ -52,7 +52,6 @@ h2:hover .anchor, h3:hover .anchor {
 }
 
 .source {
-  width: 90%;
```

Comment:
This was added in https://github.com/checkstyle/checkstyle/issues/3902 .

> https://romani.github.io/7301/site/writingjavadocchecks.html

I took one spot and zoomed in far to make the text span past the border.
![Image1](https://user-images.githubusercontent.com/5427943/70095591-2af29a80-15f3-11ea-9060-8a8ae6ac79b3.png)

It created a scrollbar on the bottom and could scroll the text, but the max text hit the border on the end.

https://github.com/checkstyle/checkstyle/pull/3998#issuecomment-287549880
Original change gave some padding between the border and the last text.

Can we add a right padding similar to the left?

Reply:
Sadly, that is not that easy because it is not possible to add css to text nodes (which the codefragments are). I can try more thorough tomorrow though.

Reply:
@jo2 Why can't you add a padding to the `source` class listed above? It is the box that needs the padding.

Reply:
@rnveach , can you explain me problem one more time.... I do not see what is wrong on big screen and on mobile for `Will lead to violation if` code block

Reply:
hmmm, I do not see this as problem, and we can improve it later on, if there possibility to do this.
I am not sure we will benefit from extra wrapping of source to something else and make source without border and border of wrap.
Please think one more time about this point.
I am ok without gap.

Reply:
I am not asking for extra wrapping, only extra padding. Text will stay on the same line as there is a scrollbar on the bottom. I am just hoping we can make some room between border and last of text.

I am fine with improving it later but I am still waiting for the message to be updated in the assertion.

Reply:
@rnveach There is a padding on the box. But the `pre` is overflowing to the right side so the padding on the right is ignored. The `pre` overflows til the border of the `source` because the `source` has `overflow: scroll` so everything which overflows the `source` will be scrolled. But we can swap the overflow from the `source` to the generated `pre` and add the padding there (see screenshot).

![image](https://user-images.githubusercontent.com/7516893/70237743-23a0cd80-1768-11ea-8507-98dbe817d437.png)

Reply:
I am not sure if you are saying this can't be fixed. We can leave it for another time.

Reply:
this is a screenshot from my lokal build. I can upload it to my github pages if you want

Reply:
Yes please upload. It is fixed in the current commit right? I just wanted to verify that is the changes I see now.

Reply:
Uploaded at https://jo2.github.io/checkstyle/

Reply:
Thanks.

Reply:
@jo2 On your version I am not seeing a scrollbar anymore or the padding on the right.

Reply:
It works in Chrome.. I'll take a look at it and try to make it work in firefox.

Reply:
I now got a solution working in chrome and firefox. Uploaded it to https://jo2.github.io/checkstyle/

Reply:
thanks @jo2 . It looks good to me. Feel free to submit another PR on it.